R/labels.R

```diff
@@ -15,3 +15,21 @@ pretty.spatial_clustering_cv <- function(x, ...) {
   res <- paste0(details$v, "-fold spatial cross-validation")
   res
 }
+
+#' Short Descriptions of spatial rsets
+#'
+#' Produce a character vector describing the spatial resampling method.
+#'
+#' @param x An `rset` object
+#' @param ... Not currently used.
+#' @return A character vector.
+#' @export pretty.spatial_block_cv
+#' @export
+#' @method pretty spatial_block_cv
+#' @rdname pretty.spatial_block_cv
```

Comment:
This should be something like this `pretty_spatialsample`, and change it for the first function too